### PR TITLE
New machines marked as NOT_WORKING 

### PR DESCRIPTION
**Pull request by clawgrip**
----------------------------------
Golden Fruits [jordigahan, ClawGrip]

### CODE REVIEW

File: src/mame/mame.lst [hunk 1]
```diff
@@ -23239,6 +23239,7 @@ missb2                          // bootleg on enhanced hardware
 
 @source:missbamby.cpp
 gwinner                         // (c) 1983 Reben
+goldfrts                        //
```
Comment (cuavas) [inline src/mame/mame.lst:23242]: Please keep this list sorted.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/mame.lst (hunk 1 above)
Comment (clawgrip) [reply src/mame/mame.lst:23242]: Fixed, thanks

File: src/mame/drivers/missbamby.cpp [hunk 2]
```diff
@@ -11,7 +11,7 @@ Known games on this hardware:
  | YES    | Lucky Player      | Cirsa              | Cirsa PCB 810702 A |
  | YES    | Miss Bamby        | Automatics Pasqual |                    |
  | YES    | Golden Winer      | Reben              |                    |
```
Comment (cuavas) [inline src/mame/drivers/missbamby.cpp:13]: Is this "Golden Winer" (something involving wine?) or "Golden Winner"?

Comment (clawgrip) [reply src/mame/drivers/missbamby.cpp]: It's a typo, fixed!

**Pull request merged**